packages/base/catalog-entry.gts

```diff
@@ -30,46 +33,12 @@ export class CatalogEntry extends CardDef {
       return new URL(this.ref.module, this[relativeTo]).href;
     },
   });
-  @field demo = contains(FieldDef);
   @field realmName = contains(StringField, {
     computeVia: function (this: CatalogEntry) {
       return this[realmInfo]?.name;
     },
   });
-  @field thumbnailURL = contains(StringField, { computeVia: () => null }); // remove this if we want card type entries to have images
-
-  get showDemo() {
-    return !this.isField;
-  }
-
-  // An explicit edit template is provided since computed isPrimitive bool
-  // field (which renders in the embedded format) looks a little wonky
-  // right now in the edit view.
-  static edit = class Edit extends Component<typeof this> {
-    <template>
-      <CatalogEntryContainer>
-        <FieldContainer @tag='label' @label='Title' data-test-field='title'>
-          <@fields.title />
-        </FieldContainer>
-        <FieldContainer
-          @tag='label'
-          @label='Description'
-          data-test-field='description'
-        >
-          <@fields.description />
-        </FieldContainer>
-        <FieldContainer @label='Ref' data-test-field='ref'>
-          <@fields.ref />
-        </FieldContainer>
-        <FieldContainer @label='Workspace Name' data-test-field='realmName'>
-          <@fields.realmName />
-        </FieldContainer>
-        <FieldContainer @vertical={{true}} @label='Demo' data-test-field='demo'>
-          <@fields.demo />
-        </FieldContainer>
-      </CatalogEntryContainer>
-    </template>
-  };
+  @field examples = linksToMany(CardDef);
```

Comment:
stub this for now. We need to consider ways to model examples for both fieldDef and cardDef. I think @demo used to be the way to do this but let's think about this in another PR. We may very well just re-include demo but we need to investigate its usage more carefully